include/powsybl/iidm/converter/xml/ExtensionXmlSerializer.hpp

```diff
@@ -28,35 +30,98 @@ namespace xml {
 class NetworkXmlReaderContext;
 class NetworkXmlWriterContext;
 
+/**
+ * An ExtensionProvider able to serialize/deserialize extensions from XML
```

Comment:
You should use {@link ExtensionProvider}, no ?

Reply:
Javadoc looks ok: I will keep it like this